docs/src/app/(docs)/api-reference/react/page.mdx

```diff
@@ -565,8 +565,8 @@ export function MultiUploader() {
     onUploadError: () => {
       alert("error occurred while uploading");
     },
-    onUploadBegin: () => {
-      alert("upload has begun");
+    onUploadBegin: ({ file }) => {
+      alert("upload has begun for", file);
```

Comment:
nit: might want to use file.name or something, since this is an alert not a console log

Reply:
file is a string, but should probably be a console log

Reply:
Ah